### PR TITLE
Add docs for attitudes_daily

### PR DESCRIPTION
**Pull request by benmiroglio**
I used the [`clients_daily` docs](https://docs.telemetry.mozilla.org/datasets/batch_view/clients_daily/reference.html) as a structural reference for these.

@sunahsuh r?

### CODE REVIEW

File: src/datasets/other/attitudes_daily/intro.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#### Contents
+
+Most Firefox surveys are point-in-time without longitudinal insights.
+The DAS is completed by ~300 Firefox users every day, allowing us to measure long term attitudinal trends combined with users' corresponding Telemetry attributes.
+
+#### Accessing the Data
+
+The `attitudes_daily` table is accessible through re:dash using the
+`Telemetry (BigQuery)` data source.
+The full table name is `moz-fx-data-shared-prod.telemetry_derived.attitudes_daily_v1`.
```
Comment (sunahsuh) [inline src/datasets/other/attitudes_daily/intro.md]: I just submitted mozilla/bigquery-etl#652 which creates a user-facing view for the table accessible as `telemetry.attitudes_daily` in STMO – in general we should prefer that view so we can make changes at the view level as necessary

File: src/datasets/other/attitudes_daily/reference.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+GROUP BY 1, 2
+```
+
+<iframe src="https://sql.telemetry.mozilla.org/embed/query/63957/visualization/163467?api_key=gzCopzybDta4t3JGkx2urB9MQa67akAehJUybVdW&" width="100%" height="570" frameborder="0" scrolling="no"></iframe>
+
+## Scheduling
+
+This dataset is updated daily via the
+[telemetry-airflow](https://github.com/mozilla/telemetry-airflow) infrastructure.
+The job runs as part of the [`attitudes_daily` DAG](https://github.com/mozilla/telemetry-airflow/blob/master/dags/attitudes_daily.py).
```
Comment (sunahsuh) [inline src/datasets/other/attitudes_daily/reference.md]: If we add a link to https://workflow.telemetry.mozilla.org/tree?dag_id=attitudes_daily folks will be able to see actual  task run status as well

File: src/datasets/other/attitudes_daily/intro.md (hunk 1 above)
Comment (benmiroglio) [reply src/datasets/other/attitudes_daily/intro.md]: 👍

File: src/datasets/other/attitudes_daily/reference.md (hunk 2 above)
Comment (benmiroglio) [reply src/datasets/other/attitudes_daily/reference.md]: 👍

**Pull request merged**